src/BinSkim.Rules/MachORules/BA5001.EnablePositionIndependentExecutable.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Composition;
+using System.Linq;
+
+using Microsoft.CodeAnalysis.BinaryParsers;
+using Microsoft.CodeAnalysis.BinaryParsers.Dwarf;
+using Microsoft.CodeAnalysis.IL.Sdk;
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.CodeAnalysis.Sarif.Driver;
+
+namespace Microsoft.CodeAnalysis.IL.Rules
+{
+    [Export(typeof(Skimmer<BinaryAnalyzerContext>)), Export(typeof(ReportingDescriptor))]
+    public class EnablePositionIndependentExecutableMachO : MachOBinarySkimmer
+    {
+        /// <summary>
+        /// BA5001
+        /// </summary>
+        public override string Id => RuleIds.EnablePositionIndependentExecutableMachO;
+
+        /// <summary>
+        /// A Position Independent Executable (PIE) relocates all of its sections at load time, including the code section,
+        ///  if ASLR is enabled in the Linux kernel (instead of just the stack/heap).  This makes ROP-style attacks more difficult.
```

Comment:
>   [](http://example.com/codeflow?start=85&length=2)

can you clean this double space?
also, in the beginning we have a space. can you also delete that?